Backport of upstream commit 51ea4bbbdce62b3e74e1defe1e57a6724987f20f
("src/CSPropDiscMaterial.cpp: std:: prefix to cout.")

--- a/src/CSPropDiscMaterial.cpp
+++ b/src/CSPropDiscMaterial.cpp
@@ -327,7 +327,7 @@
 
 bool CSPropDiscMaterial::ReadHDF5( std::string filename )
 {
-	cout << __func__ << ": Reading \"" << filename << "\"" << std::endl;
+	std::cout << __func__ << ": Reading \"" << filename << "\"" << std::endl;
 
 	// open hdf5 file
 	hid_t file_id = H5Fopen( filename.c_str(), H5F_ACC_RDONLY, H5P_DEFAULT );
